src/ostorlab/agent/message/proto/v3/report/cve/cve.proto

```diff
@@ -0,0 +1,105 @@
+syntax = "proto2";
+
+package ostorlab.agent.message.proto.v3.report.cve;
+
+message Message {
+    optional string cve_id = 1;
+    repeated Target targets = 2;
+    optional int32 cwe = 3;
+    optional string description = 4;
+    repeated Reference references = 5;
+    optional CvssScoreV3 cvss_v3_1_vector = 6;
+    optional CvssScoreV3 cvss_v3_0_vector = 7;
+    optional CvssScoreV2 cvss_v2_vector = 8;
```

Comment:
It would be nicer to group them and have a naming convention like cvss.v3_0. No need for vector in the name.